lib/iris/analysis/maths.py

```diff
@@ -332,7 +332,10 @@ def divide(cube, other, dim=None, in_place=False):
     _assert_is_cube(cube)
     other_unit = getattr(other, 'units', '1')
     new_unit = cube.units / other_unit
-    op = operator.idiv if in_place else operator.div
+    try:
+        op = operator.itruediv if in_place else operator.truediv
+    except AttributeError:
+        op = operator.idiv if in_place else operator.div
```

Comment:
1. I don't see why an AttributeError would ever get raised - Python 2 and 3 both define `operator.[i]truediv`.
2. Changing from `[i]div` to `[i]truediv` is a change of behaviour for Python 2.